src/Core/Repositories/EntityFramework/Queries/ProviderUserReadCountByOnlyOwnerQuery.cs

```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using System;
+using Bit.Core.Enums.Provider;
+using Bit.Core.Models.EntityFramework.Provider;
+
+namespace Bit.Core.Repositories.EntityFramework.Queries
+{
+    public class ProviderUserReadCountByOnlyOwnerQuery : IQuery<ProviderUser>
+    {
+        private readonly Guid _userId;
+
+        public ProviderUserReadCountByOnlyOwnerQuery(Guid userId)
+        {
+            _userId = userId;
+        }
+
+        public IQueryable<ProviderUser> Run(DatabaseContext dbContext)
+        {
+            var owners = from ou in dbContext.ProviderUsers
+                where ou.Type == ProviderUserType.ProviderAdmin &&
+                ou.Status == ProviderUserStatusType.Confirmed
+                group ou by ou.ProviderId into g
+                select new 
+                { 
+                    OrgUser = g.Select(x => new {x.UserId, x.Id}).FirstOrDefault(), ConfirmedOwnerCount = g.Count() 
```

Comment:
Won't this always return the first user in the provider?

I think you want a select many here

Reply:
I agree, something seems off here. Perhaps we could filter the select first to only include the user since we don't care about the others.

Reply:
yeah, I think updating to 
```suggestion
                    OrgUser = g.Where(x => x.Userid == _userId).Select(x => new {x.UserId, x.Id}).FirstOrDefault(), ConfirmedOwnerCount = g.Count() 
```

I'd need to test, though, linq query syntax throws me off my game

Reply:
I think I understand it now. It doesn't matter which user gets selected since we are only interested in finding ProviderUsers where we are the sole owner.

Reply:
ahhh you're right. that's too clever by half.

Converting these queries to expressions feels like the kind of thing we should do as we add queries. But If you're feeling fed up with this ticket, I'll relent.